client/css/style.css

```diff
@@ -681,19 +665,58 @@ button {
 #chat.hide-quit .quit {
 	display: none !important;
 }
-#chat .notice .type {
-	display: none;
+#chat .join .from:before {
+    font-family: FontAwesome;
+	content: "\f090";
+	color: #2ECC40;
+}
+#chat .kick .from:before {
+    font-family: FontAwesome;
+	content: "\f00d";
```

Comment:
For `/kick`, I'm not too fond of this icon, usually used for closing something and sometimes for deleting (but less and less, I notice).
I think I'd prefer `ban` (https://fortawesome.github.io/Font-Awesome/icon/ban/). If really you don't like it, the same icon as leave would work as the consequences are the same (the user leaves the channel) although I still prefer `ban` to that.
Alternatives could be https://fortawesome.github.io/Font-Awesome/icon/exclamation-triangle/, https://fortawesome.github.io/Font-Awesome/icon/exclamation-circle/, https://fortawesome.github.io/Font-Awesome/icon/exclamation/ (but these I would prefer to notify a warning within the app) and if really we want to go esoteric, these could do: https://fortawesome.github.io/Font-Awesome/icon/bolt/, https://fortawesome.github.io/Font-Awesome/icon/crosshairs/, https://fortawesome.github.io/Font-Awesome/icon/crosshairs/, https://fortawesome.github.io/Font-Awesome/icon/thumbs-o-down/, https://fortawesome.github.io/Font-Awesome/icon/frown-o/, https://fortawesome.github.io/Font-Awesome/icon/bomb/.
Meh, `ban` still conveys the most meaning, IMO :-)